### ISSUE TITLE
End spacing in column (with space between option) is not working

### ISSUE THREAD

**Issue by vardarsezgin** [FlutterFlow/flutterflow-issues#2778]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

hey!
When you have a column with multiple widgets inside and set main axis alignment as space between you can not set start & end spacing. It just acts on its own whatever value you enter.

### Expected Behavior

It should behave with whatever value you enter. For example if you set 20 it should left 20, if you set 50 it should left 50. But whatever value you set, it does acts on its own.

### Steps to Reproduce

1. create page
2. add colum
3. add 2 container or any other widgets.
4. set main axis alignment as space between
5. try to set end spacing 20,50 or whatever you want

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

CAESrBFDaEpEYjI1MFlXbHVaWEpmYW14a1lqaHljMm9TOXdzS0QwTnZiSFZ0Ymw5dFkyZDVlSFZoYWhLb0F3b1NRMjl1ZEdGcGJtVnlYMjF5TlhGcGIyd3hFdk1CQ2d4U2IzZGZaV1EwYzJKamFtVVNXZ29OU1dOdmJsODFOMmxvZW1ka014Z0lJa0k2S2dvYkNQSFRBeElKYm1WM2FXTnZibk56SUFBeUJuUnlZWE5vTWpnQkVRQUFBQUFBQURoQUlnSVFEL29EQVBJRUJBb0NDZ0R5QlFrSkFBQUFBQUFBOEQ5aUFNSUJBQkpUQ2cxVVpYaDBYM0YwYWpaelpXZzBHQUlpT3hJZkNndFRiMmhpWlhScElITnBiRG9DRUE5QUJub0tFZ2hoZFdOaGQyeG5hbm9DR0FINkF3RHlCQVFLQWdvQThnVUpDUUFBQUFBQUFQQS9ZZ0RDQVFBWUF5SWpHZ3NxQ1FrQUFBQUFBQUFVUVBvREFQSUVCQW9DQ2dEeUJRa0pBQUFBQUFBQThEODZCbTl3ZEdsdmJtSUF3Z0VBR0FFaVFnb3FDZ3NLQ1FrQUFBQUFBQUR3ZnhJS1FnSVFCbG9BWWdJUUFTSUFPQUZBQWtrQUFBQUFBQUJwUUZJQStnTUE4Z1FFQ2dJS0FQSUZDUWtBQUFBQUFBRHdQem9KWTI5dGNHOXVaVzUwWWdDS0FVVVNQd29JYlRJMmIzUnFlbW9TTStJQkpVSWZDaE1LQ25Ob2IzZEVaV3hsZEdVU0JXZDBkblZ5RWdnS0JoSUVkSEoxWlZBQ1dBS3FBZ2hzTkcwMU5IcHZjUm9DQ0FIQ0FRQVNqUUVLRUVScGRtbGtaWEpmZWpoM2FYZHdNM2tZRXlKMW1nRlFDZ0l5QVNwS0NBcFNSaEkyRWpRSURFSVpJaGNLRXdvS2MyaHZkMFJsYkdWMFpSSUZaM1IyZFhJUUFvSUJGQW9TUTI5dWRHRnBibVZ5WDJGNk9HcDJhM0o1RWdnS0JoSUVkSEoxWlNJQ0NBR3lBUTBSQUFBQUFBQUE4RDhhQWhBRWtnTUErZ01BOGdVSkNRQUFBQUFBQVBBL1lnQVN5d1lLRWtOdmJuUmhhVzVsY2w5ck5teHlaR0k0YXhMM0Fnb01VbTkzWHpnNFpIaDZhSEp6RXVRQkNnOURiMngxYlc1Zll6WnVjR2N4TnpBU1RBb05WR1Y0ZEY5bWRuZDFjSEYyTWhnQ0lqUVNHQW9HVDI1aGVXeGhNQUZBRG5vS0VnaHNkSGhrWTJkMmEzb0NHQVA2QXdEeUJBUUtBZ29BOGdVSkNRQUFBQUFBQVBBL1lnRENBUUFTV0FvTlZHVjRkRjkxYVhFeGJXcGtiQmdDSWtBU0tBb1lRblVnWlhsc1pXMGdaMlZ5YVNCaGJNU3hibUZ0WVhvdVFBWjZDaElJWldWa2QyTnVOMjM2QXdEeUJBUUtBZ29BOGdVSkNRQUFBQUFBQVBBL1lnRENBUUFZQkNJaUlnMFlCRElKQ1FBQUFBQUFBQlJBZWdJWUFmb0RBUElGQ1FrQUFBQUFBQUR3UDJJQXdnRUFFbFVLRFZSbGVIUmZZalpzWm00eWRXTVlBaUk5RWlFS0MxbHBibVVnWkdVZ1UybHNLQWM2QWhBUFFBWjZDaElJYzNKc2JHSmtNbVI2QWhnRCtnTUE4Z1FFQ2dJS0FQSUZDUWtBQUFBQUFBRHdQMklBd2dFQUdBTWlHaG9DS2dENkF3RHlCQVFLQWdvQThnVUpDUUFBQUFBQUFQQS9PZ1p2Y0hScGIyNWlBTUlCQUJnQklvSUJDaW9LQ3dvSkNRQUFBQUFBQVBCL0VncENBaEFHV2dCaUFoQUJJZ0E0QVVBQ1NRQUFBQUFBQUdsQVVnQ2FBVG9LQWpJQktqUUlERUlaSWhjS0V3b0tjMmh2ZDBSbGJHVjBaUklGWjNSMmRYSVFBb0lCRkFvU1EyOXVkR0ZwYm1WeVgyRjZPR3AyYTNKNWtnTUErZ01BOGdRRUNnSUtBUElGQ1FrQUFBQUFBQUR3UHpvSlkyOXRjRzl1Wlc1MFlnQ0tBYUlDRXBzQ0NnaHZhRFZ1ZW1GcWRCSk1JandpT2dvNENBTkNGaklVQ2hJS0NHTm9ZWFJNYVhOMEVnWTBZV05oTkhwS0JZSUJBaEFCZ2dFVUNoSkRiMjUwWVdsdVpYSmZZWG80YW5acmNubXFBZ2gwTmpWbU4yUnFiWUFGQVNyQUFRb0lNVzltYlRsemJqTVNSQkkzRUFBYUZUSVRDaEZUWTJGbVptOXNaRjlwT0dReWNuaDROaUFBS2dVSUJoRDZBVm9UQ2hGVFkyRm1abTlzWkY5cE9HUXljbmg0TnFvQ0NHdHpjMlpsYUhGdkttNEtDREI0WkROek0ycGxFbUtxQVZRS0h3b2RFaHRUYjJoaVpYUWdZbUhGbjJGeXhMRjViR0VnYzJsc2FXNWthUzRTQzBBRnNnRUdDZ1FhQWhBUUlRQUFBQUFBY0tkQUtoRWlEeElDRUFHcUFnaHBhSFkzZW1Zd2FqQUFPZ1lLQkJvQ0VBK3FBZ2gyYVhneGRHcHZOeG9DQ0FIQ0FRQVlCQ0pHSWg4WUJESWJDUUFBQUFBQUFDUkFHUUFBQUFBQUFDUkFJUUFBQUFBQUFDUkFXaFFKQUFBQUFBQUFLRUFaQUFBQUFBQUFLRUJBQWZvREFQSUZDUWtBQUFBQUFBRHdQem9NYzJWc1pXTjBWMmxrWjJWMFlnRENBUUFZQVNKVkNqY0tDd29KQ1FBQUFBQUFBUEIvRWhjU0N5Z0NNUUFBQUFBQUFDQkFRZ0lRQmxvQVlnSVFBU0lBT0FGQUFra0FBQUFBQUFCcFFGSUFXZ3M1QUFBQUFBQUFBQUJBQXZvREFQSUZDUWtBQUFBQUFBRHdQem9TUkhKdmNHUnZkMjVmT0Y5UGNIUnBiMjV6WWdEQ0FRQT0=

### Context

was trying to set end spacing in column

### Visual documentation

[https://github.com/FlutterFlow/flutterflow-issues/assets/103498910/e88a9222-e807-4c6e-ae21-43b5633fa85d](https://github.com/FlutterFlow/flutterflow-issues/assets/103498910/e88a9222-e807-4c6e-ae21-43b5633fa85d)

### Additional Info

*No response*

### Environment

```markdown
- FlutterFlow version: 4.1.34 Prod and 4.1.40 beta both
- Platform: MacOS and Web
- Browser name and version: Chrome 123.0.6312.124
- Operating system and version affected: Sonoma 14.4 (23E214)
```

**Comment by paulperez-dev** [comment]
Hi @vardarsezgin, thanks for your report! Your report code is wrong but I was able to reproduce the issue. I will send it to the team so they can take a look.

**Comment by vardarsezgin** [comment]
Hey @paulperez-dev , i probably copied something else by mistake. Thanks

**Comment by takashiff** [comment]
Hi @vardarsezgin, thanks for the bug report! We've marked this as a feature request since currently this is intended behavior due to the way our code generation works. However, I completely understand that this effect may not be what our users might expect, so we've created a ticket internally to keep track of this. I'm going to go ahead and mark this as closed for now, but just know that this is something we are actively tracking. Thanks!
